### PR TITLE
Destination snowflake: Depend on published cdk artifact

### PR DESCRIPTION
**Pull request by edgao**
code changes in https://github.com/airbytehq/airbyte/pull/36396; forgot to publish cdk >.>

### CODE REVIEW

File: airbyte-integrations/connectors/destination-snowflake/build.gradle [hunk 1]
```diff
@@ -3,9 +3,9 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.23.18'
+    cdkVersionRequired = '0.26.2'
```
Comment (jbfbell) [inline airbyte-integrations/connectors/destination-snowflake/build.gradle:6]: `0.27.2` ?

**Review by jbfbell — CHANGES_REQUESTED** [review]

**Review by jbfbell — APPROVED** [review]

**Comment by alafanechere** [comment]
@edgao [I ran the test on a pre-release version of airbyte-ci ](https://github.com/airbytehq/airbyte/actions/runs/8434576668/job/23098087586) which is likely fixing the problem you had in using the latest CDK . The airbyte-ci PR is here: https://github.com/airbytehq/airbyte/pull/36480

**Pull request merged**